tests/Amazon.CloudWatch.EMF.IntegrationTests/MetricsLoggerIntegrationTest.cs

```diff
@@ -115,11 +115,14 @@ private Dimension GetDimension(string name, string value)
             return res;
         }
 
-        private void LogMetric(String metricName)
+        private void LogMetric(String metricName, bool isHighResolution = false)
```

Comment:
Was this added to test toggling of integration tests for high resolution? Can we remove this? This will be always false right.

Reply:
done